src/tensors/factorizations.jl

```diff
@@ -440,6 +435,8 @@ function tsvd!(t::TensorMap;
     else
         truncerr = abs(zero(scalartype(t)))
         W = S(dims)
+        # TODO: do we really want this behaviour? this changes the arrows of the S legs, but
+        # only sometimes?
```

Comment:
Good question. I think the idea was that if either U or V can be interpreted as simply a unitary basis transform on the domain or the codomain, we make them as such. But this can actually be extended even if U or V are isometries. As soon as either N1 or N2 are 1, S could inherit the arrow from there. Not sure if that is something useful.

Reply:
Yeah, in general it is a bit of a strange question. I think I would prefer it if we just fix the arrow, i.e. result in a non-dual space. I mostly don't like that the behaviour is dependent on the truncation scheme, which I think might be confusing/inconsistent.

Reply:
That is definitely true. I also don't see why that was necessary. The only case where I could see that it is potentially useful to have other arrows is when either the domain and/or codomain of `t` have just one space, and you want `U` or `V` to represent a linear map on that space. But this works as well in the truncated case. 

But I am certainly fine with just having a fixed convention, so feel free to remove these lines.